src/app/pages/task-calendar/rsync/rsync-form/rsync-form.component.ts

```diff
@@ -189,20 +194,21 @@ export class RsyncFormComponent {
         type: 'textarea',
         name: 'rsync_extra',
         placeholder: T('Extra options'),
-        tooltip: T('Add any other <a\
+        tooltip: T('Additional <a\
                     href="https://rsync.samba.org/ftp/rsync/rsync.html"\
-                    target="_blank">rsync(1)</a> options. The "*"\
-                    character must be escaped with a backslash (\\*.txt)\
-                    or used inside single quotes(\'*.txt\').'),
+                    target="_blank">rsync(1)</a> options to include.<br>\
+                    Note: The "*" character <b>must</b> be escaped with\
+                    a backslash (\\*.txt) or used inside single quotes\
+                    (\'*.txt\').'),
       }, {
         type: 'checkbox',
         name: 'rsync_enabled',
         placeholder: T('Enabled'),
         tooltip: T('Unset to disable the rsync task without deleting it.\
-                    When the <a href="..//docs/services.html#rsync"\
-                    target="_blank">Rsync service</a> is OFF, the rsync\
-                    task continues to look for the server unless this\
-                    option is unset.'),
+                    When the <a href="../docs/services.html#rsync"\
+                    target="_blank">Rsync service</a> is <b>OFF</b>,\
```

Comment:
We do not have services set to OFF.  They can be running or not running.  Additionally, I can't tell what the rest of this tooltip means.

Reply:
This tooltip is wrong. Replaced with the standard Enabled tooltip.
Enable this rsync task. Unset to disable this rsync task without deleting it.'